gangplank/cmd/gangplank/commonflags.go

```diff
@@ -16,4 +33,9 @@ func init() {
 	specCommonFlags.StringSliceVar(&generateSingleRequires, "singleReq", []string{}, "artifacts to require")
 	specCommonFlags.StringVarP(&cosaSrvDir, "srvDir", "S", "", "directory for /srv; in pod mount this will be bind mounted")
 	jobspec.AddKolaTestFlags(&cosaKolaTests, specCommonFlags)
+
+	user, _ := user.Current()
+	sshFlags.StringVar(&minioSshRemoteHost, "forwardMinioSSH", containerHost(), "forward and use minio to ssh host")
```

Comment:
```suggestion
	sshFlags.StringVar(&minioSshRemoteHost, "forwardMinioSSH", "", "forward and use minio to ssh host")
```

`containerHost()` returns `user@host.com` whereas I think we want `minioSshRemoteHost` to just be `host.com`. Maybe we should just leave it blank by default.

Reply:
Actually, no. The Minio forwarding code accepts both, and uses the `who@` as the user if not defined.

Reply:
There was some weird combination I was hitting earlier where my ssh connections were trying `builder@fedora@host.com`. not sure what was happening there and I thought this was part of the problem. Maybe not.

Reply:
You're correct. `containerHost()` is stripping off the envVar. I'll fix that.

Reply:
Nice find. There were two bugs in that logic tree. 

Fixed and repushed.